frontend/src/algolia.css

```diff
@@ -252,6 +252,7 @@
 
   .ais-Panel-header {
     @apply mb-2 text-xs font-bold uppercase;
+    @apply sticky top-0 bg-gray-50;
```

Comment:
This additional styling may be merged into the line above. I don't see any need of applying them separately.

Reply:
I agree.